code/controllers/Processes/pipenet.dm

```diff
@@ -1,14 +1,16 @@
-var/global/list/datum/pipe_network/pipe_networks = list()
-var/global/list/obj/machinery/atmospherics/atmos_machines = list()
+var/list/datum/pipe_network/pipe_networks = list()
```

Comment:
And to anyone that might be confused about that having global there in the var declaration would make it clear.

Absolute pathing is also redundant, but there are steps we take to make code easier to understand.